manage.py

```diff
@@ -115,7 +115,6 @@ def init_db(skip_create_db=False, test_db=False, force=False):
         db_uri = frontend.create_app().config['SQLALCHEMY_DATABASE_URI']
 
     if not skip_create_db:
-        init_postgres(db_uri)
         create_extension(db_uri)
```

Comment:
I just noticed that we also create the extension with this entrypoint mount:
https://github.com/metabrainz/critiquebrainz/blob/master/docker/docker-compose.dev.yml#L18

This means that we can also remove `create_extension` and `skip_create_db`

Reply:
check for uses in the code of `--skip-create-db`, there are some in the test scripts, and also in the documentation